packages/devtools_app/lib/src/screens/vm_developer/vm_script_display.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'vm_developer_common_widgets.dart';
+import 'vm_object_model.dart';
+
+/// A widget for the object inspector historyViewport displaying information
+/// related to script objects in the Dart VM.
+class VmScriptDisplay extends StatelessWidget {
+  const VmScriptDisplay({
+    required this.script,
+  });
+
+  final ScriptObject script;
+
+  @override
+  Widget build(BuildContext context) => VmObjectDisplayBasicLayout(
+        object: script,
+        generalDataRows: _scriptDataRows(script),
+      );
```

Comment:
nit: prefer using brackets for returns that are more than one line, and especially for flutter build methods

Reply:
Done